sl/symbin.cc

```diff
@@ -271,7 +271,8 @@ bool handleNoOp(
     const CodeStorage::TOperandList &opList = insn.operands;
 
     // this allows functions with 0 or 1 parameters
-    if (opList.size() < 2 || 3 < opList.size()) {
+    if ((opList.size() < 2 || 3 < opList.size())
```

Comment:
Could the condition be simplified as `(opList.size() < 2 || 4 < opList.size())`?